src/web/app/pages-static/terms-page/terms-page.component.html

```diff
@@ -1,22 +1,26 @@
-<main>
-  <h1 class="color-orange">
-    Terms of Use
-  </h1>
+<main class="padding-top-10px">
+  <div class="row">
+    <h1 class="color-orange">
+      Terms of Use
+    </h1>
+  </div>
   <img src="assets/images/terms.png">
-  <p>
-    By using <b>TEAMMATES</b>, you agree to the following terms of use:
-  </p>
-  <p>
-    <b>Quality of service</b>: We take pride in our work and we shall do our best to provide good service to users. However, the service is provided "AS IS" and WITHOUT ANY WARRANTY.
-    You use TEAMMATES at your own discretion and risk and you will be solely responsible for any damages that may arise from such use.
-  </p>
-  <p>
-    <b>Changes to the service</b>: We strive to improve TEAMMATES continuously and provide its services free for as long as we can. However, TEAMMATES services may change or be terminated at our discretion.
-  </p>
-  <p>
-    <b>Security and privacy of data</b>: Our data are stored in Google servers and are protected by the same security mechanisms that protect Google data.
-    However, you are advised not to store in TEAMMATES any 'sensitive' data such as credit card numbers and passwords.<br>
-    We do not use TEAMMATES data for research or share TEAMMATES data with third parties.
-    While TEAMMATES maintainers (tenured developers) have some access to system data for performance monitoring and troubleshooting purposes, only TEAMMATES administrators have access to secured and identifiable data.
-  </p>
+  <div class="row">
```

Comment:
I assume the extra div is being added for better horizontal padding? If we change the padding in the main content instead, this would likely not be needed.